crates/bevy_ecs/src/storage/sparse_set.rs

```diff
@@ -323,6 +323,12 @@ impl ComponentSparseSet {
         }
     }
 
+    pub fn shrink_to_fit(&mut self) {
```

Comment:
This should probably have a doc comment as it's actually usable outside of `bevy_ecs`.